### PR TITLE
docs(identity): add logging configuration page detailing options available

### PR DESCRIPTION
**Pull request by Ben-Sheppard**
Closes https://github.com/camunda-cloud/identity/issues/1740

## What is the purpose of the change
There have been recent requests from clients based on the Identity logging configuration that could have been solved by better documentation around the options available.

This PR introduces a dedicated page detailing the options for logging in Identity, including:
* The framework used
* The supplied configuration options
* A method for supplying your own logging configuration

## Are there related marketing activities
There are not :)

## When should this change go live?
This change is not part of a wider co-ordinated effort and I would not consider it urgent, therefore it is fine to be released in the natural release cadence.

## PR Checklist

- [x] My changes apply to an already released version, and I have added them to the relevant `/versioned_docs` directory, or they are not for an already released version.
- [x] My changes apply to future versions, and I have added them to the main `/docs` directory, or they are not for future versions.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

File: docs/self-managed/identity/user-guide/configuration/configure-logging.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+## Configuring logging
+
+The Identity component uses the [Log4j2](https://logging.apache.org/log4j/2.x/) framework to control
+the log level and log format.
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (dlavrenuek) [inline docs/self-managed/identity/user-guide/configuration/configure-logging.md:17]: The included configuration file here is missing any description. We should add something like "following is the log4j configuration included in Identity".

File: docs/self-managed/identity/user-guide/configuration/configure-logging.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+| `IDENTITY_LOG_FILE_PATTERN`       | _See the [Log4j2 pattern layout docs](https://logging.apache.org/log4j/2.x/manual/layouts.html#PatternLayout) for possible placeholders._                                |
+| `IDENTITY_LOG_FILE_ROTATION_DAYS` | _See the [Log4j2 time-based triggering policy -> interval](https://logging.apache.org/log4j/2.x/manual/appenders.html#timebased-triggering-policy) for possible values._ |
+| `IDENTITY_LOG_FILE_ROTATION_SIZE` | _See the [Log4j2 size-bsed triggering policy](https://logging.apache.org/log4j/2.x/manual/appenders.html#sizebased-triggering-policy) for possible values._              |
+
+</TabItem>
+</Tabs>
+
+### Providing your own logging configuration
+
+The default logging configuration file can be overridden by setting the following variable:
```
Comment (dlavrenuek) [inline docs/self-managed/identity/user-guide/configuration/configure-logging.md:108]: This sentence just tells the user to set the `LOGGING_CONFIG`, which does not answer following questions that users will probably have:
- where is my logging configuration?
- how do I add my logging configuration to Identity?
- are there any rules for file location?

May be something like?

```
You can provide your own configuration by mounting the configuration file to the Identity container and setting it's path in the following variable:
```

File: versioned_docs/version-8.1/self-managed/identity/user-guide/configure-logging.md [hunk 3]
```diff
@@ -1,22 +1,104 @@
...
+                name="File"
+                fileName="${env:IDENTITY_LOG_FILE_NAME:-logs/identity.log}"
+                filePattern="${env:IDENTITY_LOG_FILE_NAME_PATTERN:-${LOG_FILE_NAME_PATTERN}}"
+                append="true">
+            <PatternLayout pattern="${env:IDENTITY_LOG_FILE_PATTERN:-${LOG_FILE_PATTERN}}"/>
+            <Policies>
+                <TimeBasedTriggeringPolicy interval="${env:IDENTITY_LOG_FILE_ROTATION_DAYS:-1}"/>
+                <SizeBasedTriggeringPolicy size="${env:IDENTITY_LOG_FILE_ROTATION_SIZE:-50 MB}"/>
+            </Policies>
+        </RollingFile>
```
Comment (dlavrenuek) [inline versioned_docs/version-8.1/self-managed/identity/user-guide/configure-logging.md:44]: @Ben-Sheppard if this is referenced as what we include per default, then `File` logger is not included on 8.1

**Review by dlavrenuek — CHANGES_REQUESTED** [review]
Great addition 🏅 Some small suggestions from my side

**Review by dlavrenuek — APPROVED** [review]
@christinaausley I have fixed some small errors in the last commit and it looks from my perspective

**Comment by christinaausley** [comment]
> @christinaausley I have fixed some small errors in the last commit and it looks from my perspective

Awesome. I'll merge and launch this with the release today. We can always iterate moving forward 👍

**Pull request merged**